bus/client.go

```diff
@@ -262,6 +262,29 @@ func (c *Client) AddRenewedContract(contract rhpv2.ContractRevision, totalCost t
 	return
 }
 
+// AncestorContracts returns any ancestors of a given active contract.
+func (c *Client) AncestorContracts(fcid types.FileContractID, minStartHeight uint64) (contracts []ArchivedContract, err error) {
+	values := url.Values{}
+	values.Set("minStartHeight", fmt.Sprint(minStartHeight))
+	err = c.c.GET(fmt.Sprintf("/contracts/%s/ancestors?"+values.Encode(), fcid), &contracts)
+	return
+}
+
+// HistoricalContractSpending returns the spending of the contract and all the
+// contracts in its ancestry that have been formed at or after minStartHeight.
+func (c *Client) HistoricalContractSpending(contract Contract, minStartHeight uint64) (ContractSpending, error) {
+	ancestors, err := c.AncestorContracts(contract.ID, minStartHeight)
```

Comment:
client methods should not wrap other client methods. Either `AncestorContracts` should return `ContractSpending` directly, or (preferably) this function should be moved to whatever package needs it.